activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -1149,6 +1148,18 @@ def find_from_target?
         def exec_queries
           load_target
         end
+
+        def respond_to_missing?(method, _)
+          scope.respond_to?(method) || super
```

Comment:
Should not `_` be passed to `respond_to?`? Also I don't think `super` does something here.